### PR TITLE
Transparency page v1

### PR DESCRIPTION
**Pull request by jparnaudo**
This PR addresses [this issue](https://www.notion.so/dcl-dao/Add-Transparency-Page-f2e361c5192044a59948609f017cfb4a)

How it looks for desktop:

# Our mission section
image

## Current Balance
Data for each token in each wallet only shows if there is a > 0 amount:

Tokens in 3 wallets
image

Tokens in 2 wallets
image

No tokens
image

## Monthly Income/Expenses
Percentage shows color depending on positive/negative values.
image

# Funding
image

If there are more than 5 proposals, View more label appears.
image

# Members
image

Each team has an info link with a description
image

# Mobile

iPhone SE
image

image

iPad
image

image

### CODE REVIEW

**Comment by jparnaudo** [comment]
Styling revamp:

image

image


Mobile:

iPhone
image

image


iPad
image

**Comment by jparnaudo** [comment]
Changes after Squad Demo:

Changed Monthly Income/Expense titles and sub
image

Removed total budget and progress bar
image

Removed Organizational Chart link, changed buttons texts and icons
image

Checked Popups For Desktop/Mobile:

Desktop
image

iPhone SE
image

Kind of in the limit, but readable. What do you think @yemel 
image

image

**Review by yemel — APPROVED** [review]
LGTM, tested locally.

File: Pulumi.website-dao-prd.yml [hunk 1]
```diff
@@ -14,13 +14,11 @@ config:
   website-dao:GATSBY_BUY_MANA_URL: https://account.decentraland.org/
   website-dao:GATSBY_BUY_NAME_URL: https://builder.decentraland.org/names
   website-dao:GATSBY_DECENTRALAND_API: https://decentraland.org
+  website-dao:GATSBY_DCL_DATA_API: https://data.decentraland.vote/
   website-dao:GATSBY_DEFAULT_CHAIN_ID: "1"
   website-dao:GATSBY_DISCOURSE_API: https://forum.decentraland.org
   website-dao:GATSBY_DISCOURSE_CATEGORY: "13"
   website-dao:GATSBY_DISCOURSE_USER: dao
-  website-dao:GATSBY_DURATION_GRANT_TIER4: "1209600"
```
Comment (2fd) [inline Pulumi.website-dao-prd.yml:21]: This shouldn't be removed

**Review by 2fd — CHANGES_REQUESTED** [review]

File: Pulumi.website-dao-prd.yml (hunk 1 above)
Comment (yemel) [reply Pulumi.website-dao-prd.yml:21]: those are duplicated on master:

https://github.com/decentraland/governance/blob/master/Pulumi.website-dao-prd.yml#L44

**Pull request merged**